src/lib/transaction_logic/test/transaction_logic.ml

```diff
@@ -239,3 +239,41 @@ let%test_module "Test transaction logic." =
                  Transaction_status.equal txn.command.status Applied ) )
             (run_zkapp_cmd ~fee_payer ~fee ~accounts txns) )
   end )
+
+(* This module tests Inputs.Stack *)
+let%test_module "Test stack module" =
+  ( module struct
+    module Stack = Transaction_logic.For_tests.Stack (Int)
+
+    let%test_unit "Ensure pop works on non-empty list." =
+    Quickcheck.test ~trials 
+    (let open Quickcheck in
+    let open Generator.Let_syntax in
```

Comment:
Can these two lines not be combined with` let open Quickcheck.Let_syntax `?

Reply:
I'll keep it as for a couple of reasons: 
1. I need to open `Quickcheck` already to use `Generator` functions, so might as well leave the first line as is.
2. In the case that `Generator.let_syntax` diverges from `Quickcheck.let_syntax` in a later version of the library, I'd rather stick with the former in this context.